### PR TITLE
no more forcing of helvetica

### PR DESCRIPTION
**Pull request by tank157**
[Pivotal Card](https://www.pivotaltracker.com/story/show/118262197) + 2 other cover letter cards.

This change to stop the forcing of helvetica is necessary so that the cover letters can render the address information in an OCR friendly font for the USPS.

Will need to test NVRF forms and VBM forms in krakenstein before merging this.

### CODE REVIEW

**Comment by tank157** [comment]
Assigning to @sduckett

File: src/form_works/render.clj [hunk 1]
```diff
@@ -51,7 +38,6 @@
                                     .getDefaultResources
                                     .getFonts)]
           (when default-fonts (.putAll page-fonts default-fonts))
```
Comment (sduckett) [inline src/form_works/render.clj]: Will all of the forms have functional default fonts?

Comment (tank157) [inline src/form_works/render.clj]: I'm not really sure if they will or not, there is a standard set of available fonts including Helvetica though. I believe this portion of code was in the cribbed code for field flattening referenced in the comment though, I didn't want to change it without cause.

**Comment by sduckett** [comment]
LGTM, @tank157.

**Pull request merged**